pkg/cli/cli.go

```diff
@@ -70,8 +70,8 @@ type cli struct {
 	pluginsFromOptions map[string][]plugin.Base
 	// Default plugins injected by options.
 	defaultPluginsFromOptions map[string][]plugin.Base
-	// A mapping of plugin name to version passed by to --plugins.
-	cliPluginKeys map[string]string
+	// A plugin key passed to --plugins on invoking 'init'.
+	cliPluginKey string
```

Comment:
I feel a little confused, maybe because I forgot some context.
Only one plugin can be passed to `--plugins`?

Reply:
For now yeah. Having multiple plugin arguments to `--plugins` only makes sense if you can chain them together (phase 2).